src/setupTools.js

```diff
@@ -217,6 +217,15 @@ export default function setupTools(store) {
     { name: 'ZoomTouchPinch' },
     { name: 'StackScrollMouseWheel' },
     { name: 'StackScrollMultiTouch' },
+    {
+      name: 'ArrowAnnotate',
+      mouseButtonMasks: [1],
```

Comment:
If you remove this line, I'll merge this PR.
Specifying `mouseButtonMasks` is supposed to be used to set the initial active/bound tools for the viewport.
I get that the other config doesn't follow this pattern, just calling it out here.

Reply:
Yep, there was a lot of cargo culting here. Thanks Danny!